### PR TITLE
[docs][ybm] CLI roles, users, api-key

### PR DESCRIPTION
**Pull request by ddhodge**
Add role, user, and api-key commands

@netlify /preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-role.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## Commands
+
+### create
+
+Create a custom role.
+
+| Flag | Description |
+| :--- | :--- |
+| --role-name | Required. Name for the role. |
+| --permissions | Required. Permissions for the role, provided as key value pairs. Permissions are made up of a resource type and an operation group.<br>Arguments:<ul><li>resource-type - the resource to which the permission applies</li><li>operation-group - the operation that the permission allows on the resource</li></ul>Both resource-type and operation-group are mandatory. Specify multiple permissions by using multiple --permissions arguments. Use `ybm role describe --role-name admin` to view a list of all permissions. |
```
Comment (harshitha-yb) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-role.md:56]: > Use `ybm role describe --role-name admin` to view a list of all permissions

Do we mean "Use `ybm permission list` to view a list of all available permissions"?

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-user.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Use the `user` resource to invite and manage YugabyteDB Managed account [users](../../../../cloud-admin/manage-access/).
+
+## Syntax
+
+```text
+Usage: ybm user [command] [flags]
+```
+
+## Examples
+
```
Comment (harshitha-yb) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-user.md:24]: nit: Can we have an example for inviting a user as well?

**Review by harshitha-yb — COMMENTED** [review]
Looks good! Just a couple of minor comments/questions.
Also we recently added support for API Key management as well from CLI 
Do we already have documentation for that?

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-role.md (hunk 1 above)
Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-role.md:56]: Ya. That was added after I first wrote this and this was my workaround

**Review by harshitha-yb — APPROVED** [review]
Any update on the docs for API Keys?

**Comment by ddhodge** [comment]
I just added api-key to this PR

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-api-key.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Use the `api-key` resource to create and manage YugabyteDB Managed account [API keys](../../../managed-apikeys/).
+
+## Syntax
+
+```text
+Usage: ybm api-key [command] [flags]
+```
+
+## Examples
+
```
Comment (harshitha-yb) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-reference/managed-cli-api-key.md:24]: Can we have an example for create API Key as well?

**Review by harshitha-yb — APPROVED** [review]
Looks good!

**Pull request merged**